src/exchange/depthChart/depthchart.ts

```diff
@@ -351,7 +351,7 @@ function sellsAfterForSellOffer(remaining: BigNumber, price: BigNumber, sells: O
   let remainingUnused = true;
   const sellsResult = sells.reduce(
     (result: Offer[], offer: Offer) => {
-      if (remainingUnused && price.isEqualTo(offer.price)) {
+      if (remainingUnused && price.eq(offer.price)) {
```

Comment:
what's the difference? According to docs its the same(?) https://mikemcl.github.io/bignumber.js/